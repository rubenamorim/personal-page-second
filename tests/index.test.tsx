import React from 'react';
import { render } from 'react-testing-library';

import App from 'pages';

it('renders the homepage', () => {
    const { getByText, container } = render(<App />);

    expect(getByText('Hello ðŸ‘‹')).toBeInTheDocument();
    expect(container).toMatchSnapshot();
});
