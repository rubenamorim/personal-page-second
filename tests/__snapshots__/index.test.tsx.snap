// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the homepage 1`] = `
<div>
  <main
    class="css-1rzbih8-Main e8842ay0"
  >
    Hello ðŸ‘‹
    <section
      class="css-r9ew2p-Section enmh62p0"
    >
      <div
        class="css-uz0s87-Triangle enmh62p1"
      />
      <svg
        class="css-qd7e9n-StyledGithub enmh62p2"
        viewBox="0 0 24 24"
      >
        <path
          d="M12 0C5.374 0 0 5.373 0 12c0 5.302 3.438 9.8 8.207 11.387.599.111.793-.261.793-.577v-2.234c-3.338.726-4.033-1.416-4.033-1.416-.546-1.387-1.333-1.756-1.333-1.756-1.089-.745.083-.729.083-.729 1.205.084 1.839 1.237 1.839 1.237 1.07 1.834 2.807 1.304 3.492.997.107-.775.418-1.305.762-1.604-2.665-.305-5.467-1.334-5.467-5.931 0-1.311.469-2.381 1.236-3.221-.124-.303-.535-1.524.117-3.176 0 0 1.008-.322 3.301 1.23A11.509 11.509 0 0 1 12 5.803c1.02.005 2.047.138 3.006.404 2.291-1.552 3.297-1.23 3.297-1.23.653 1.653.242 2.874.118 3.176.77.84 1.235 1.911 1.235 3.221 0 4.609-2.807 5.624-5.479 5.921.43.372.823 1.102.823 2.222v3.293c0 .319.192.694.801.576C20.566 21.797 24 17.3 24 12c0-6.627-5.373-12-12-12z"
        />
      </svg>
      <svg
        class="css-1crhw9-StyledArrow enmh62p3"
        viewBox="0 0 512 512"
      >
        <path
          d="M41.037 340.689a8.486 8.486 0 0 1-8.211-6.367 8.5 8.5 0 0 1 5.368-10.131l47.232-16.829a9.212 9.212 0 0 1 1.771-.431l175.874-22.665-187.493 9.186c-4.606.165-8.552-3.267-8.884-7.859a8.513 8.513 0 0 1 3.665-7.627l-37.103-4.957a8.497 8.497 0 0 1-7.287-9.551c.626-4.639 4.949-7.913 9.542-7.283l40.509 5.426 197.604-4.033 193.192-18.379c4.768-.563 8.813 2.984 9.261 7.644.438 4.667-2.976 8.813-7.651 9.252l-14.242 1.359a8.389 8.389 0 0 1 1.425 3.063 8.48 8.48 0 0 1-4.493 9.616l-18.055 8.887a8.912 8.912 0 0 1-2.671.813l-79.998 10.313-307.975 50.437a7.903 7.903 0 0 1-1.38.116z"
        />
        <path
          d="M329.317 354.699a8.496 8.496 0 0 1-7.319-4.162 8.5 8.5 0 0 1 2.76-11.489L448 260.805c-74.71-34.093-124.071-56.376-131.821-59.51a9.038 9.038 0 0 1-4.287-2.354 8.498 8.498 0 0 1-2.038-3.25c-1.534-2.388-8.075-12.112-13.962-20.708a8.5 8.5 0 0 1 .58-10.338 8.532 8.532 0 0 1 10.147-2.089l171.972 83.606a8.469 8.469 0 0 1 4.758 7.154 8.421 8.421 0 0 1-1.045 4.6 8.514 8.514 0 0 1 3.797 6.771 8.498 8.498 0 0 1-4.41 7.76l-148.303 81.21a8.44 8.44 0 0 1-4.071 1.042z"
        />
        <path
          d="M302.913 385.505a8.497 8.497 0 0 1-7.526-4.542c-2.181-4.161-.571-9.286 3.572-11.475l161.062-84.477-162.737-143.654a8.496 8.496 0 0 1-.745-11.988c3.1-3.515 8.472-3.855 11.987-.746l171.97 151.813a8.471 8.471 0 0 1 2.802 7.378 8.472 8.472 0 0 1-4.476 6.499L306.85 384.526a8.424 8.424 0 0 1-3.937.979z"
        />
      </svg>
      <span
        class="css-tdsr8e-Text enmh62p4"
      >
        Follow the progress
      </span>
    </section>
  </main>
</div>
`;
